.gitignore

```diff
@@ -187,7 +190,8 @@ $RECYCLE.BIN/
 .LSOverride
 
 # Icon must end with two \r
-Icon


```

Comment:
I don't understand why this has been changed. `^M` is a carriage return from Windows IIRC. Were you at a Windows machine?

Reply:
yes